deploy/deploy.sh

```diff
@@ -33,6 +33,9 @@ JAZZER_COORDINATES=$1
 [ ! -f "${JAZZER_JAR_PATH}" ] && \
   fail "JAZZER_JAR_PATH does not exist at '$JAZZER_JAR_PATH'"
 
+"${JAZZER_JAR_PATH}" --version 2>&1 | grep '^Jazzer v' || \
```

Comment:
Run this with the `java` executable as well?

Reply:
Done (using the hermetic JDK for style points :-))

Reply:
haha, nice :smile: